src/beta/resources/end_shipper.js

```diff
@@ -0,0 +1,28 @@
+import T from 'proptypes';
+import base from '../../resources/base';
+
+export const propTypes = {
```

Comment:
Are we able to consolidate and reuse this across `Address` and `EndShipper`?

Reply:
yes, we can, added a `base_address` file